fabfile/adage_server.py

```diff
@@ -191,8 +191,11 @@ def import_data_and_index():
         CONFIG['data']['gene_history_file'])
     run('python manage.py import_gene_network  %s "Ensemble ADAGE 300"' %
         CONFIG['data']['gene_network_file'])
-    run('python manage.py import_node_gene_network %s "Ensemble ADAGE 300"' %
-        CONFIG['data']['node_gene_network_file'])
+    run('python manage.py create_or_update_participation_type'
+        ' "High weight genes" "Genes that most strongly influence the'
+        ' activation of a signature."')
```

Comment:
Nice! I wonder if the second part of that, which is in the data migration file that automatically adds the ``High weight genes`` ParticipationType if no other PT object exists in the database, would be useful to have here. It reads "and we have found that they often reveal the underlying process or processes captured by the signature."" What do you think?